src/uiTest/scala/com/virtuslab/gitmachete/uitest/UIScenarioSuite.scala

```diff
@@ -0,0 +1,59 @@
+package com.virtuslab.gitmachete.uitest
+
+import com.virtuslab.gitmachete.testcommon.BaseGitRepositoryBackedIntegrationTestSuite
+import com.virtuslab.gitmachete.testcommon.SetupScripts.SETUP_README_SCENARIOS
+import org.junit._
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.virtuslab.ideprobe._
+import org.virtuslab.ideprobe.dependencies.IntelliJVersion
+
+object UIScenarioSuite
+  extends RunningIntelliJPerSuite
+    with IdeProbeFixture
+    with RunningIntelliJFixtureExtension {
+
+  lazy val intelliJVersion: IntelliJVersion = {
+    val version = sys.props.get("ui-test.intellij.version")
+      .filterNot(_.isEmpty).getOrElse(throw new Exception("IntelliJ version is not provided"))
+    // We're cheating here a bit since `version` might be either a build number or a release number,
+    // while we're always treating it as a build number.
+    // Still, as of ide-probe 0.26.0, even when release number like `2020.3` is passed as `build`, UI tests work just fine.
+    IntelliJVersion(build = version, release = None)
+  }
+
+  override protected def baseFixture: IntelliJFixture = {
+    // By default, the config is taken from <class-name>.conf resource, see org.virtuslab.ideprobe.IdeProbeFixture.resolveConfig
+    fixtureFromConfig().withVersion(intelliJVersion)
+  }
+
+}
+
+@RunWith(classOf[JUnit4])
+class UIScenarioSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_README_SCENARIOS) {
```

Comment:
Actually! are we going to run this suite on each CI build, just as the other suite?

Reply:
idk yet. That would make the build runs last longer

Reply:
Oh! what's the problem on the CI?